test/Driver/arch-specific-macros.f

```diff
@@ -0,0 +1,15 @@
+! RUN: %flang -target aarch64 -v -x f77 -### -c - 2>&1 >/dev/null | FileCheck --check-prefix=AARCH64 %s
```

Comment:
`-v` not needed

Reply:
This test file is useless anyway. It uses `%flang` as it assumes that pull request https://github.com/flang-compiler/clang/pull/20 was pulled into frontend driver. Now as flang moved to release_60 and affected configuration file lit.cfg migrated to lit.cfg.py, referenced pull request won't apply any longer.